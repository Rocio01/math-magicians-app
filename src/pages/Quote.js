import React from 'react';

const Quote = () => (
  <div>

    <p>
      If I were again beginning my studies, I would follow the advice of
      Plato and start with mathematics.
      <small> Galileo Galilei </small>
    </p>
  </div>
);

export default Quote;
