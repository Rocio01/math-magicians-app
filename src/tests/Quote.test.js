import renderer from 'react-test-renderer';
import Quote from '../pages/Quote';

test('matches home snapshot', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});