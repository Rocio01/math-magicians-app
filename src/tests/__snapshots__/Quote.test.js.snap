// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches home snapshot 1`] = `
<div
  className="p-5 m-5"
>
  <p
    className="p-5 m-5"
  >
    If I were again beginning my studies, I would follow the advice of Plato and start with mathematics.
    <small>
       -Galileo Galilei 
    </small>
  </p>
</div>
`;
